specifyweb/frontend/js_src/lib/components/FormPlugins/collectionRelData.ts

```diff
@@ -59,7 +59,8 @@ export const processColRelationships = async (
 export async function fetchOtherCollectionData(
   resource: SpecifyResource<CollectionObject>,
   relationship: string,
-  formatting: string | undefined
+  formatting: string | undefined,
+  isInFormEditor: boolean
```

Comment:
(optional)

Not the biggest fan of calling the parameter `isInFormEditor`. 
Is there a possibility this function will be used outside of rendering the CollectionRelPlugin? 

The name is descriptive for the current uses/callers of `fetchOtherCollectionData` for _why_ the parameter is needed but does not provide information regarding what the parameter does. 

Specifically, callers of the function will likely need to inspect the body of the function to determine what `isInFormEditor` does and how it alters the function's behavior (especially since it is required). 

If it is likely this will be used outside of rendering the plugin, maybe something like `muteIncorrectCollectionError`?

Reply:
good point - improved